src/Testing/CrawlerTrait.php

```diff
@@ -338,7 +338,7 @@ protected function seeJsonContains(array $data)
             json_decode($this->response->getContent(), true)
         ));
 
-        foreach (array_sort_recursive($data) as $key => $value) {
+        foreach (array_sort_recursive(json_decode(json_encode($data), true)) as $key => $value) {
```

Comment:
I agree. I didn't see a Laravel helper function that performs the same functionality (converting all objects, and nested objects to an array recursively).  I mainly just wanted to point out there is an issue, and this is how it can be fixed.  I will take no issue if you want to close this request and implement a more "eloquent" solution.

Currently, you can't have an object anywhere in the data that you pass into this function, because it's keys will not be sorted.  All objects in the response data are converted to arrays, so their keys are sorted.